frontend/src/components/AccountReceiveTable.vue

```diff
@@ -337,7 +337,7 @@
 
 <script lang="ts">
 import { computed, getCurrentInstance, defineComponent, onMounted, PropType, ref } from '@vue/composition-api';
-import { date, copyToClipboard } from 'quasar';
+import { copyToClipboard } from 'quasar';
```

Comment:
_(arbitrary comment location to start a thread)_

> Should we surface stealth address? Should I add whether it was sent in advance mode or add any other data potentially missing?

Good call, it would be nice to also track:
1. ENS/CNS name AND address that resolved to, or just address if no ENS was given (since a user may enter an ENS name, and what it resolves to at the time of their tx may be different than what it resolves to in the future
2. Was advanced mode enabled (as a boolean)
3. Was the "Send using recipient's standard public key" box checked (as a boolean)

Items 2 and 3 I don't think we need to show the user, but they are good to have available for debugging, etc.

Reply:
Okay yes I can add those checkboxes